packages/data-stores/src/help-center/index.ts

```diff
@@ -9,10 +9,16 @@ import type { SelectFromMap, DispatchFromMap } from '../mapped-types';
 
 export function register(): typeof STORE_KEY {
 	registerStore< State >( STORE_KEY, {
+		// eslint-disable-next-line @typescript-eslint/ban-ts-comment
+		// @ts-ignore Until thunks are used correctly
 		actions,
```

Comment:
It looks like it still doesn't like the use of thunks in actions, maybe?  I had to add the ts-ignore